docs/src/basics/autodiff.md

```diff
@@ -2,7 +2,7 @@
 
 !!! note
     
-    We support all backends supported by DifferentiationInterface.jl. Please refer to
+    We support ForwardDiff.jl, FiniteDiff.jl and PolyesterForwardDiff.jl(PolyesterForwardDiff only for collocation methods) via DifferentiationInterface.jl. Please refer to
```

Comment:
My only concern is that this is a breaking change

Reply:
Actually, it's not, the current BoundaryValueDiffEq.jl only supports ForwardDiff.jl and FiniteDiff.jl(restricted by SparseDiffTools.jl), so this instead could unlock other backends(at least add PolysterForwardDiff support)